ecl2df/gruptree.py

```diff
@@ -1,13 +1,10 @@
-#!/usr/bin/env python
-"""
-Extract GRUPTREE information from an Eclipse deck
-
-"""
+"""Extract GRUPTREE information from an Eclipse deck"""
```

Comment:
I liked the previous format better (without the empty line). Have used it for all docstrings in subscript-internal. Should I change?

Reply:
It is the previous format that has the extra empty line, not sure if I understand the comment.